src/platform/forms/save-in-progress/actions.js

```diff
@@ -58,6 +58,74 @@ export const PREFILL_STATUSES = {
   unfilled: 'unfilled',
 };
 
+const validateFetchInProgressFormsErrors = status => {
+  // check if typeof status is a string, return that string
+  if (
+    typeof status === 'string' &&
+    Object.values(LOAD_STATUSES).includes(status)
+  ) {
+    return status;
+  }
+
+  // check if it is an array
+  if (Array.isArray(status?.errors) && status?.errors.length > 0) {
+    switch (status.errors[0].status) {
```

Comment:
Why are we relying on something other than [the actual response status](https://developer.mozilla.org/en-US/docs/Web/API/Response/status) to provide us with a status code? That seems less reliable.